seismic/xcorqc/xcorqc.py

```diff
@@ -84,40 +84,44 @@ def taper(tr, taperlen):
     return tr
 # end func
 
-def whiten(x, sr, fmin=None, fmax=None):
+def whiten(a, sampling_rate, window_freq=0):
+    # frequency step
+    npts = a.shape[0]
+    deltaf = sampling_rate / npts
 
-    if not fmin: fmin = 0
-    if not fmax: fmax = sr/2.
+    ffta = np.fft.rfft(a)
 
-    # apply hanning window
-    l = x.shape[0]
-    h = signal.hann(l)
-    xf = rfft(h*x)
+    # smooth amplitude spectrum
```

Comment:
It would be good to have a bit more explanation in the following 10 lines or so of code.

Reply:
Just pushed some comments through